internal/cliwrapper/cliwrapper.go

```diff
@@ -86,9 +86,9 @@ func (p *cliWrapper) KillAndClean(containerName string) error {
 	cmdKill := p.getPodmanCmd("kill", containerName)
 	p.logger.Debugf("Killing with command %v", cmdKill.Args)
 	if err := cmdKill.Run(); err != nil {
-		p.logger.Warningf("failed to kill pod %s, probably the execution terminated earlier", containerName)
+		p.logger.Infof("failed to kill pod %s, probably the execution terminated earlier", containerName)
 	} else {
-		p.logger.Warningf("successfully killed container %s", containerName)
+		p.logger.Infof("successfully killed container %s", containerName)
```

Comment:
Is this even worth logging? Should it be a debug severity instead?

Reply:
Hm. It could go either way. If it gets here, it means it was still running. I think it's better for it to finish on its own. But debug instead of no logging for sure.